README.md

```diff
@@ -26,17 +27,27 @@ update with the `--no-cache` flag.
 composer update --no-cache
 ```
 
-If you want to be sure you are getting the latest commits when developing, clearing composer cache, deleting the folders and re-running composer update seems to be a solid approach:
+If you want to be sure you are getting the latest commits when developing,
+clearing composer cache, deleting the folders and re-running composer update
+seems to be a solid approach:
 
 ```bash
 
-rm -rf web/profiles/contrib/ web/modules/contrib/ web/themes/contrib/; 
+rm -rf web/profiles/contrib/ web/modules/contrib/ web/themes/contrib/;
 composer clear-cache; composer update --with-dependencies --no-cache;
 lando start;
 lando drush si localgov -y;
 
 ```
 
+If you run into [memory limit errors](https://getcomposer.org/doc/articles/troubleshooting.md#memory-limit-errors)
+when running Composer commands, prefix the commands with `COMPOSER_MEMORY_LIMIT=-1`.
+For example, to install the project run:
+
+```bash
+COMPOSER_MEMORY_LIMIT=-1 composer create-project --stability dev localgovdrupal/localgov-project MY_PROJECT
+```
+
```

Comment:
I was trying to deploy Localgov in Azure the other day and had to add this COMPOSER_MEMORY_LIMIT=-1 while doing composer install.  The default 1.6GB memory simply wasn't enough for LocalGov!  I wanted to thank you and Finn at that time for adding this in the README :)